core/components/atoms/chip/_stories_/index.story.tsx

```diff
@@ -0,0 +1,76 @@
+import * as React from 'react';
+import { select, boolean, text } from '@storybook/addon-knobs';
+import Chip from '../Chip';
+import { updateKnob } from '@/utils/storybookEventEmitter';
+import { action } from '@storybook/addon-actions';
+
+export const all = () => {
+
+  const type = select(
+    'type', ['action', 'selection', 'input'],
+    'action'
+  );
+
+  const icon = text(
+    'icon', 'assessment'
+  );
+  const label = text(
+    'label',
+    'ChipLabel',
+  );
+  let clearbutton;
+  if (type !== 'action') {
+    clearbutton = boolean(
+      'clearbutton', true
+    );
+  }
+
+  const disabled = boolean(
+    'disabled',
+    false
+  );
+  let selected;
+  if (type === 'selection') {
+    selected = boolean(
+      'selected',
+      false
+    );
+    updateKnob('label', 'Selection');
+  }
+  if (type === 'action') {
+    updateKnob('label', 'Action');
+  }
+  if (type === 'input') {
+    updateKnob('label', 'Input');
+  }
+  const open = boolean('open', true);
```

Comment:
@stutirao  Please remove `open` from knobs as it is not a prop for the component it creates confusion.